css/default.css

```diff
@@ -863,7 +855,7 @@ hr {
 	background: White;
 }
 #submit {
-	background-color: #778899; /*var(--primary-color3); LightSlateGray */
+	background-color: #778899; /*var(--primary-color3); LightSlateGray
```

Comment:
This seems to leave the comment open until closing some place later. Might fix an error, but hides bits in between.

Reply:
The comment opens at line 843 and is closed at line 864.  

The validator did not like the close at 858 followed by another close at 864.  The extra open at 858 appears to be a non-issue.

Should this entire section be deleted?

Reply:
Yes, up to and including `End of search form` etc this can be deleted, thanks.